python-clusters/create-gke-cluster/cluster.json

```diff
@@ -31,19 +31,28 @@
             "label": "Networking",
             "description": "<ul><li>For VPC-native clusters, leave IP ranges empty to let GKE automatically assign them.</li></ul>"
         },
+        {
+            "name": "isInheritedFromDSSHost",
```

Comment:
Should be called `inheritFromDSSHost`

`isBlabla` is a wording reserved for accessors, not for the actual boolean variable

Reply:
Fixed.